### PR TITLE
fix TorchEngine stuck when benchmarking with `tp>1`

### PR DESCRIPTION
**Pull request by grimoire**
Pool is daemontic, create submodule in daemontic process is not allowed.

### CODE REVIEW

**Comment by AllentDan** [comment]
我这边会和之前一样卡住。@HIT-cwh 可以试下OK不？

**Comment by AllentDan** [comment]
May resolve conflicts.

**Comment by lvhan028** [comment]
May fix the conflict

**Review by AllentDan — APPROVED** [review]
LGTM

File: benchmark/profile_torch_generation.py [hunk 1]
```diff
@@ -123,8 +123,11 @@ def profile_throughput(model_path: str, concurrency: int, input_seqlen: int,
     from lmdeploy.messages import PytorchEngineConfig
     from lmdeploy.pytorch.engine import Engine
 
-    tm_model = Engine(model_path, PytorchEngineConfig(model_name='llama',
-                                                      tp=tp))
+    tm_model = Engine(
```
Comment (lvhan028) [inline benchmark/profile_torch_generation.py:126]: profile_torch_throughput.py 要修改吗？

Comment (grimoire) [reply benchmark/profile_torch_generation.py:126]: https://github.com/InternLM/lmdeploy/blob/a22ddab9fde114d5c4e03cd65bc0869b0f05d414/benchmark/profile_torch_throughput.py#L69

main 里已经修了

**Comment by irexyc** [comment]
https://github.com/InternLM/lmdeploy/blob/main/lmdeploy/pytorch/config.py#L8C1-L16

这个地方应该有问题，torch_dtype 是GenerationConfig的属性，默认是None，所以default没什么用

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**